src/views/admin/extend/rewards.tpl

```diff
@@ -27,9 +27,9 @@
 							<div class="well inline-block">
 								<label for="condition">[[admin/extend/rewards:condition-then]]</label><br />
 								<select name="rid" data-selected="{active.rid}">
-									<!-- BEGIN rewards -->
+									<!-- BEGIN ../../rewards -->
```

Comment:
What about `conditions` and `conditionals`? Do they need to be modified the same way?

Reply:
No, those don't refer to scoped variables like active.rewards does.